api/src/main/java/org/openmrs/module/appointments/service/AppointmentsService.java

```diff
@@ -38,6 +38,9 @@ public interface AppointmentsService {
     @Authorized({VIEW_APPOINTMENTS, MANAGE_APPOINTMENTS})
     List<Appointment> getAllAppointmentsReminder(String hours);
 
+    @Transactional
+    @Authorized({VIEW_APPOINTMENTS, MANAGE_APPOINTMENTS})
```

Comment:
Why does one need MANAGE_APPOINTMENTS privilege to get appointments?  This doesn't seem right, even if the other service methods already make this mistake.

Reply:
MANAGE_APPOINTMENTS covers all - view as well as add/edit. At least that the impression I had.

Reply:
Oh, interesting.  By default, this is an _or_ operation.  I had just assumed this was an _and_.